rust/cubestore/cubestore/src/scheduler/mod.rs

```diff
@@ -568,9 +568,9 @@ impl SchedulerImpl {
             || chunks.len()
             > self.config.compaction_chunks_count_threshold() as usize
             // TODO config
-            || in_memory_chunks.len() > 100
-            || min_in_memory_created_at.map(|min| Utc::now().signed_duration_since(min).num_seconds() > 60).unwrap_or(false)
-            || max_created_at.map(|min| Utc::now().signed_duration_since(min).num_seconds() > 600).unwrap_or(false)
+            || in_memory_chunks.len() > self.config.compaction_in_memory_chunks_max_length() as usize
```

Comment:
`as usize` is unnecessary here because the `compaction_in_memory_chunks_max_length` function already returns `usize`